graphics/glslang/Portfile

```diff
@@ -3,10 +3,11 @@
 PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
+PortGroup           legacysupport 1.1
 
 github.setup        KhronosGroup glslang 12.2.0
 github.tarball_from archive
-revision            0
+revision            1
```

Comment:
this supposedly fixes build failures only - correct? If so, there shouldn't be a need to increase the "revision".

Reply:
Okay, I'll remove the revbump then.